api/api/accounts/views.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import generics
+
+from .models import User
+from .serializers import UserSerializer
+
+
+class UsersListView(generics.CreateAPIView):
```

Comment:
Only the AdminUsers should be able to call this API.

We don't want just anyone to create users. On SignUp the users are created. And if needed this API will be used to create Users only be the admin.

Tests will change accordingly.